src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const bzzOrNotToBzz = (firstNumber) => {
+    const helperFunction = (secondNumber) => {
```

Comment:
```suggestion
    return (secondNumber) => {
```
you can return it right away